build.gradle

```diff
@@ -91,3 +93,17 @@ tasks {
         dependsOn(ktlintFormat)
     }
 }
+
+static def getIdeVersions(Project project) {
+    def ideVersions = project.hasProperty('ideVersions') ? Arrays.asList(project.ideVersions.split(',')) :
```

Comment:
isn't there a way to say ['2019.3.5', 'latest'] or something like that? no one will remember to update this

Reply:
there's no 'latest' option, i tried... but there's no need - every stable version goes through eap beforehand - which means the ci will catch it.
this is how the old gh action meant to work - i think its good enough.
i can make the eap job scheduled for every week or so (in addition to push event), so that even if we don't open a pr for a long time we'd still catch compatibility issues.
wdyt?

Reply:
Well, the EAP shoul dnot fail the PR, only notify us so we can prepare.

Maybve you can do something similiar to eap - download latest releast and verify from disk?

Reply:
that's what the eap job does - i extracted it to a nightly job